Project.toml

```diff
@@ -3,7 +3,11 @@ uuid = "140ffc9f-1907-541a-a177-7475e0a401e9"
 version = "0.5.0"
 
 [deps]
+AbstractOperators = "d9c5613a-d543-52d8-9afd-8f241a8c3f1c"
```

Comment:
Is this needed anywhere?

Reply:
Yes, in fact it’s already listed among the test dependencies a little below here